cql3/lists.cc

```diff
@@ -166,7 +166,7 @@ lists::value::equals(shared_ptr<list_type_impl> lt, const value& v) {
             [t = lt->get_elements_type()] (const bytes_opt& e1, const bytes_opt& e2) { return t->equal(*e1, *e2); });
 }
 
-std::vector<bytes_opt>
+const std::vector<bytes_opt>&
```

Comment:
This can potentially be a source of a very nasty bug. Did you make sure reference returned from get_elements() never outlives the value object? This comment relates to the whole patch not just to the lists::value. It's just impossible to comment on a commit in github.

Reply:
Yes